developer/src/kmc-keyboard-info/src/font-family.ts

```diff
@@ -16,7 +16,12 @@ export async function getFontFamily(source: Uint8Array) {
   }
 
   const buffer = Buffer.from(source);
-  const font = await ttfMeta.promise(buffer);
+  let font = null;
+  try {
+    font = await ttfMeta.promise(buffer);
+  } catch(e) {
+    return null;
```

Comment:
It might be nice to capture the error string in the compiler error?

Reply:
I'll open a new fix for this.